cmd/gaia/app/app.go

```diff
@@ -25,16 +23,43 @@ import (
 	"github.com/tendermint/tendermint/libs/log"
 )
 
-const (
-	appName = "GaiaApp"
-)
+const appName = "GaiaApp"
 
-// default home directories for expected binaries
 var (
-	DefaultCLIHome  = os.ExpandEnv("$HOME/.gaiacli")
+	// default home directories for gaiacli
+	DefaultCLIHome = os.ExpandEnv("$HOME/.gaiacli")
+
+	// default home directories for gaiad
 	DefaultNodeHome = os.ExpandEnv("$HOME/.gaiad")
+
+	// The ModuleBasicManager is in charge of setting up basic,
+	// non-dependant module elements, such as codec registration
+	// and genesis verification.
+	BasicGaiaApp sdk.ModuleBasicManager
```

Comment:
Maybe we call this something else? Like GaiaAppInit?

Reply:
I think `Init` is misleading because it's used for all sorts of stuff. previously I've been calling it `Mbm`... open to further suggestions

Reply:
I'll have to think on this...

Reply:
renamed it "ModuleBasics" for now